coral/src/app/context-provider/AuthProvider.test.tsx

```diff
@@ -84,75 +82,36 @@ describe("AuthProvider.tsx", () => {
     });
   });
 
-  describe("does not render the auth provider with given children when auth user is SUPERADMIN", () => {
-    beforeEach(async () => {
-      getAuthMock.mockReturnValue({ mockAuthUser, userrole: "SUPERADMIN" });
-      customRender(
-        <AuthProvider>
-          <ChildComponent />
-        </AuthProvider>,
-        {
-          queryClient: true,
-        }
-      );
-
-      await waitForElementToBeRemoved(screen.getByText("Loading Klaw"));
-    });
-
-    afterEach(() => {
-      jest.resetAllMocks();
-      cleanup();
-    });
-
-    it("does not returns context provider with given children", () => {
-      const childElement = screen.queryByTestId("auth-provider-child");
-      expect(childElement).not.toBeInTheDocument();
-    });
-
-    it("shows a dialog informing superadmin they can not access coral", async () => {
-      const dialog = await screen.findByRole("dialog", {
-        name: "You're currently logged in as superadmin.",
-      });
-
-      expect(dialog).toBeVisible();
-    });
-  });
-
-  describe("render the auth provider with given children when auth user is SUPERADMIN and feature flag is enabled", () => {
-    beforeEach(async () => {
-      setupFeatureFlagMock(
-        FeatureFlag.FEATURE_FLAG_SUPER_ADMIN_ACCESS_CORAL,
-        true
-      );
-      getAuthMock.mockReturnValue({ mockAuthUser, userrole: "SUPERADMIN" });
-      customRender(
-        <AuthProvider>
-          <ChildComponent />
-        </AuthProvider>,
-        {
-          queryClient: true,
-        }
-      );
-
-      await waitForElementToBeRemoved(screen.getByText("Loading Klaw"));
-    });
-
-    afterEach(() => {
-      jest.resetAllMocks();
-      cleanup();
-    });
-
-    it("returns context provider with given children", () => {
-      const childElement = screen.getByTestId("auth-provider-child");
-      expect(childElement).toBeVisible();
-    });
-
-    it("does not show dialog informing superadmin they can not access coral", () => {
-      const dialog = screen.queryByText(
-        "You're currently logged in as superadmin."
-      );
-
-      expect(dialog).not.toBeInTheDocument();
-    });
-  });
+  //   describe("renders an auth provider with given children when auth user is SUPERADMIN", () => {
```

Comment:
@programmiri Would you be able to look at this at some stage?
I haven't been able to get the check to work for the superadmin so I might be missing something, but I made the updates in the commented out test I think are required.

Reply:
Sure!

Reply:
You can remove those tests all together. 

They covered the case that an user is SUPERADMIN & ff is not enabled, which has shown a "No Access" site -> this is not needed anymore, since we grant access without the flag. 

The other covered that a SUPERADMIN with an enabled FF sees the child component, which is the same as for USERs.

Both cases do not apply anymore. We only need to cover the path that the page auth user is loading and that the user is available, both are covered.

Reply:
Thank you!